catalog-rest-service/src/main/resources/json/schema/metadataIngestion/workflow.json

```diff
@@ -5,6 +5,32 @@
   "description": "OpenMetadata Ingestion Framework definition.",
   "type": "object",
   "definitions": {
+    "azureSSOConfig": {
+      "description": "Azure SSO client security configs.",
+      "type": "object",
+      "properties": {
+        "secretKey": {
+          "description": "Azure SSO client secret key",
+          "type": "string"
+        },
+        "authority": {
+          "description": "Azure SSO Authority",
+          "type": "string"
+        },
+        "clientId": {
+          "description": "Azure Client ID.",
+          "type": "string"
+        },
+        "scopes": {
+          "description": "Azure Client ID.",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "additionalProperties": false,
+      "required": ["secretKey"]
```

Comment:
@ayush-shah - So, clientId, scopes, authority should also be required fields right ?

Reply:
Yes - will add that.